packages/Button/Button.jsx

```diff
@@ -18,7 +18,7 @@ const preventDisabling = ({ disabled, ...props }) => {
 
   return props
 }
-
+const fullwidthButton = '100% !important'
```

Comment:
This can be removed.

Reply:
Hi @theetrain,
I have no permission to code commit on tds-core repository
I am getting error :
remote: Permission to telus/tds-core.git denied to sudhirraivns0103.
fatal: unable to access 'https://github.com/telus/tds-core/': The requested URL returned error: 403